apple-docs/AppleBranchingScheme.md

```diff
@@ -1,38 +1,45 @@
 # Apple's branching scheme for llvm-project
 
-There are currently three namespaces for branches on
-[github.com/apple/llvm-project](https://github.com/apple/llvm-project):
+This document explains the various important branches in
+[apple/llvm-project](https://github.com/apple/llvm-project),
+how they relate to one another, and how they correspond to branches in the
+[apple/swift](https://github.com/apple/swift) repository.
 
- 1. `llvm.org/*`, for forwarded branches from
-    [github.com/llvm](https://github.com/llvm/llvm-project);
- 2. `apple/*`, for standalone downstream content; and
- 3. `swift/*`, for downstream content that depends on
-    [Swift](https://github.com/apple/swift).
+These are the most important branches to know about:
 
-## Forwarded branches from [github.com/llvm](https://github.com/llvm/llvm-project)
+- [next](https://github.com/apple/llvm-project/tree/next)
+  is aligned with Swift's
+  [next](https://github.com/apple/swift/tree/next) branch.
+- The current stabilization branch
+  ([stable/20220421](https://github.com/apple/llvm-project/tree/stable/20220421)
+  at time of writing; check
+  [update-checkout-config.json](https://github.com/apple/swift/blob/main/utils/update_checkout/update-checkout-config.json)
+  for up-to-date info) is aligned with Swift's
+  [main](https://github.com/apple/swift/tree/main) branch.
+- `swift/*` branches are aligned with the corresponding Swift branch without
```

Comment:
Branches with a swift prefix are no longer used. They existed to differentiate between branches with swift-lldb content. I'm fine with not mentioning them or moving them to the historical trivia section.

Reply:
Looking at <https://github.com/apple/llvm-project> now, I still see `swift/release/5.7` in active use, among other `swift/*` branches.  Can you clarify what you mean?  Will this be a new practice starting with Swift 5.8?

Reply:
Good point. I guess we kept the `swift` prefix there for consistency, even thought there's not really any ambiguity, expect maybe with the upstream LLVM releases. Feel free to ignore this!